components/formats-api/src/loci/formats/meta/MetadataConverter.java

```diff
@@ -1021,6 +1021,10 @@ private static void convertImages(MetadataRetrieve src, MetadataStore dest,
           try {
             Boolean bigEndian = src.getPixelsBinDataBigEndian(i, q);
             dest.setPixelsBinDataBigEndian(bigEndian, i, q);
+            NonNegativeLong length = src.getPixelsBinDataLength(i, q);
```

Comment:
Should each pair be in a separate try/catch block so they always get called if the previous set failed for any reason?